React/Views/RCTView.m

```diff
@@ -445,17 +590,28 @@ - (BOOL)performAccessibilityAction:(NSString *) name
   return NO;
 }
 
-#if !TARGET_OS_OSX // ]TODO(macOS ISS#2323203)
+#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
 - (BOOL)accessibilityActivate
 #else // [TODO(macOS ISS#2323203)
 - (BOOL)accessibilityPerformPress
 #endif // ]TODO(macOS ISS#2323203)
 {
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+  if ([self isAccessibilityEnabled] == NO) {
```

Comment:
This method is plagued by early returns. Would be awesome if we could fix those up instead of adding one more to it :)